### ISSUE TITLE
SBT launcher not found

### ISSUE THREAD

**Issue by nh13** [travis-ci/travis-ci#9191]
Build: https://travis-ci.org/fulcrumgenomics/fgbio/builds/338089955

It looks like it is trying to download http://repo.scala-sbt.org/scalasbt/maven-releases/org/scala-sbt/sbt-launch/1.0.4/sbt-launch.jar, which I am able to download locally, but fails in travis-ci's environment.

Another build for the same PR completed here, downloading the sbt JAR just fine: https://travis-ci.org/fulcrumgenomics/fgbio/builds/337263419

Thanks for any help you can offer in advance.

**Comment by BanzaiMan** [comment]
Could you try restarting it?

**Comment by nh13** [comment]
@BanzaiMan I tried restarting the build a few times.  No joy.

**Comment by BanzaiMan** [comment]
https://github.com/paulp/sbt-extras/blob/master/sbt#L263 the `sbt` doesn't tell us *why* the download failed, so it's hard to figure out why it failed, and how to fix it. I tend to think this is a network issue of some kind. Or some rate limiting on the server.

**Comment by gvolpe** [comment]
I experienced the same issue with `sbt 1.1.1`. and Scala `2.11.2`: https://travis-ci.org/gvolpe/http4s-tracer/jobs/358838313#L464

However, I got a different error with Scala `2.12.4`: https://travis-ci.org/gvolpe/http4s-tracer/jobs/358838312#L646

Interestingly, after swapping the order of the Scala versions this error disappeared but now I get a BAD GATEWAY error for both versions and finally they time out. The command is just an `sbt clean test` that runs in a couple of seconds on my machine: https://travis-ci.org/gvolpe/http4s-tracer/jobs/358876497#L645

With swapping the order I mean I had defined:

```
scala:
  - 2.12.4
  - 2.11.12
```
And I changed it for:

```
scala:
  - 2.11.12
  - 2.12.4
```

Any ideas?
